lib/cisco_node_utils/cmd_ref/show_version.yaml

```diff
@@ -50,18 +50,19 @@ last_reset_reason:
   get_value: "rr_reason"
 
 last_reset_time:
-  _exclude: [ios_xr]
+  _exclude: [N3k, ios_xr]
   kind: string
   default_value: ''
   get_value: "rr_ctime"
 
 system_image:
-  nexus:
-    get_value: "kick_file_name"
+  get_value: "kick_file_name"
+  N5k: &sys_img_isan
+    get_value: "isan_file_name"
+  N6k: *sys_img_isan
+  N7k: *sys_img_isan
```

Comment:
@robert-w-gries , YAML anchors and alias: See: https://github.com/cisco/cisco-network-node-utils/blob/develop/lib/cisco_node_utils/cmd_ref/README_YAML.md#yaml-anchors-and-aliases